tests/e2e/lib.sh

```diff
@@ -307,6 +308,30 @@ export_central_basic_auth_creds() {
     ci_export "ROX_PASSWORD" "$ROX_PASSWORD"
 }
 
+deploy_optional_e2e_components() {
+    info "Installing optional components used in E2E tests"
+
+    if [[ "${INSTALL_COMPLIANCE_OPERATOR:-false}" == "true" ]]; then
+        install_the_compliance_operator
+    else
+        info "Skipping the compliance operator install"
+    fi
+}
+
+install_the_compliance_operator() {
+    info "Installing the compliance operator"
+
+    # ref: https://docs.openshift.com/container-platform/4.13/security/compliance_operator/compliance-operator-installation.html
+
+    oc create -f "${ROOT}/tests/e2e/yaml/compliance-operator/namespace.yaml"
+    oc create -f "${ROOT}/tests/e2e/yaml/compliance-operator/operator-group.yaml"
+    oc create -f "${ROOT}/tests/e2e/yaml/compliance-operator/subscription.yaml"
+
+    wait_for_object_to_appear openshift-compliance deploy/compliance-operator
+
+    oc get csv -n openshift-compliance
```

Comment:
I didn't know about the OS `ClusterServiceVersion`, but now that I've looked it up, I'm curious why we are "getting" this info, but not doing anything with it?

Is it just to log the info to Standard Out, in case we need to troubleshoot a build?

(sorry if this is a noob question)

Reply:
I added it because the install instructions told me to. So maybe it'll be useful to future debug.